package.json

```diff
@@ -130,9 +130,11 @@
     "jw-paginate": "^1.0.4",
     "lodash": "^4.17.21",
     "polished": "^4.2.2",
+    "prop-types": "^15.0.0",
```

Comment:
You have to run `yarn install` before commiting so that `yarn.lock` is updated too.

Reply:
Do I have to run that with yarn classic? I think it's time to move on. So I want to try Yarn 4 If ur ready.
There was no problem when I ran adminjs depending on the package of Yarn 4 + PnP.